ops/_private/harness.py

```diff
@@ -3176,6 +3176,36 @@ def autostart_services(self, timeout: float = 30.0, delay: float = 0.1):
                 self._service_status[name] = pebble.ServiceStatus.ACTIVE
 
     def replan_services(self, timeout: float = 30.0, delay: float = 0.1):
+        for name, check in self._render_checks().items():
+            if check.startup != pebble.CheckStartup.DISABLED:
```

Comment:
Nit: maybe `if check.startup == DISABLED: continue` to avoid excess indentation?